assets/js/modules/analytics-4/components/common/PropertySelect.js

```diff
@@ -74,6 +74,16 @@ export default function PropertySelect( props ) {
 		select( MODULES_ANALYTICS_4 ).getPropertyID()
 	);
 
+	const isResolvingProperties = useSelect( ( select ) => {
+		if ( hasModuleAccess === false || ! accountID ) {
+			return false;
+		}
+
+		return select( MODULES_ANALYTICS_4 ).isResolving( 'getProperties', [
+			accountID,
+		] );
+	} );
```

Comment:
Why not merge this into the `useSelect` below since it's only used as part of the `isLoading` calculation? It also wouldn't be necessary to do if `isDisabled`.